test/features/offline_mode/woker_queue_test.dart

```diff
@@ -0,0 +1,73 @@
+@TestOn('vm')
+
+import 'dart:io';
+
+import 'package:core/utils/app_logger.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jmap_dart_client/jmap/core/state.dart';
+import 'package:tmail_ui_user/features/caching/config/hive_cache_config.dart';
```

Comment:
Your code is great, but your file name is misspelling :)

Reply:
Please give your suggestions